samples/tab-channel-group-quickstart/ts/src/components/App.tsx

```diff
@@ -3,8 +3,7 @@
 
 import './App.css';
 import * as microsoftTeams from "@microsoft/teams-js";
-import { BrowserRouter as Router, Route } from "react-router-dom";
-
+import { BrowserRouter,Route ,Routes} from 'react-router-dom';
```

Comment:
Add proper spacing in between comma like below 
{ BrowserRouter, Route, Routes }